src/simmi/exam1/DifferenceAge.java

```diff
@@ -0,0 +1,54 @@
+/*Coding Exam - 1 : 18th April 2021
+
+Program 2 : 
+WAP to provide the difference between a Younger and an elder member of the family. Age is provided in the form of an array.
+
+Sample Input: 7 12 56 62 25
+
+Sample Output: 55*/
+
+package simmi.exam1;
+
+import java.util.Scanner;
+
+public class DifferenceAge {
+	int getMaxAge(int[] age) {
+		int outputMax = age[0];
+		for(int index = 0; index<age.length;index++) {
+			if(age[index]>outputMax) {
+				outputMax = age[index];
+			}
+		}
+		return outputMax;
+	}
+	
+	int getMinAge(int[] age) {
```

Comment:
Min and max can handle in single method with single loop, You should use this approach